build.sh

```diff
@@ -43,7 +43,7 @@ update_changelog ()
 
 create_dist ()
 {
-    python setup.py sdist
+    python3 setup.py sdist
```

Comment:
I'm fine with being explicit, but needing to specify the python version to me means virtualenv/conda isn't being used? which it should be... yeah, just a thought.

Reply:
Pytorch is 0.7 GB, Bert uncased 1.2 GB -> I'm using global installs for heavy libs (mobile phone hotspot).